Bot-Development/Discord-Telegram-Bots/level-4/moderationbot_AdyaArchita/handlers/moderation.py

```
@@ -0,0 +1,363 @@
+    await asyncio.sleep(5)
+    await context.bot.delete_message(chat_id=chat_id, message_id=response_msg.message_id)
+
+
+async def profanity_filter(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    """Auto-moderation filter for banned words."""
+    if not update.message or not update.message.text: return
+    
+    # Do not moderate admins
+    try:
+        chat_admins = await context.bot.get_chat_administrators(update.effective_chat.id)
+        if update.effective_user.id in {admin.user.id for admin in chat_admins}: return
```

Ruff (0.14.2)

Multiple statements on one line (colon)

(E701)